src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
```suggestion
public class ColorSupplier {
    private final Random random = new Random();
    
    public String getRandomColor() {
        Color color = Color.values()[random.nextInt(Color.values().length)];
```

Reply:
Done. But could you please explain, why it is better to use additional private final random variable, instead of my decision? One of the points of Common Mistakes said “Don’t create temporary variables if you can directly return value from the method”. So, I decided to realize my decision without additional variable. 

Color color = Color.values()[new Random().nextInt(Color.values().length)];

Maybe I misunderstood some things/rules. I don’t have strong feeling of correct decision.